packages/sdks/scripts/remove-mitosis-resolutions.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var path = require('path');
+
+const packageJson = require('../package.json');
+delete packageJson.resolutions['@builder.io/mitosis-cli'];
```

Comment:
If this runs in CI only, maybe we could replace this script with a `jq` invocation?

Reply:
Short answer was that I never used `jq` before. But thanks for pushing me to try it! Pretty neat tool to add to my repertoire :) https://github.com/BuilderIO/builder/pull/961/commits/1524eaa264a6a75c9d3363f0128fc6b01412bbf2